### ISSUE TITLE
DaSiamRPN performance on OTB-100

### ISSUE THREAD

**Issue by hellbell** [foolwood/DaSiamRPN#19]
Hey, this repository is really cool!
Here's my question.
I tried DaSiamRPN on VOT2017 and OTB-100.
The VOT2017 results are the same as your paper (ECCV18), but the OTB-100 results are shown like this,

> Mean Precision: 0.846642, Mean IOU AUC: 0.639728

which is a little bit lower than your paper's performance. 
If you have some hyper-parameter for OTB set, could you let me know them?

Thanks!

**Comment by ucasqcz** [comment]
> Hey, this repository is really cool!
> Here's my question.
> I tried DaSiamRPN on VOT2017 and OTB-100.
> The VOT2017 results are the same as your paper (ECCV18), but the OTB-100 results are shown like this,
> 
> > Mean Precision: 0.846642, Mean IOU AUC: 0.639728
> 
> which is a little bit lower than your paper's performance.
> If you have some hyper-parameter for OTB set, could you let me know them?
> 
> Thanks!
I only get AUC 0.557 on OTB2013, still not find the bug on evaluation.

**Comment by foolwood** [comment]
@hellbell @ucasqcz 
This [repository](https://github.com/foolwood/DaSiamRPN) is used to reproduce the results of VOT2018 challenge (EAO:0.3827).
The [SiamRPNBIG.model](https://drive.google.com/file/d/1-vNVZxfbIplXHrqMHiJJYWXYWsOIvGsf/view?usp=sharing) focuses on the bounding box regression for achieving high accuracy on VOT2018. This model is different from the model used in ECCV version.

Our `SiamRPNOTB.model` is released on [Google Drive](https://drive.google.com/open?id=1BtIkp5pB6aqePQGlMb2_Z7bfPy6XEj6H).
The AUC on OTB2015 is 0.655 and the speed is 200FPS with GTX 1080, which is the **fastest** Siamese Tracker.

Thanks for your question.

**Comment by csg1111** [comment]
I  tried on OTB100,but i got a bad result:
test/OTB2015/SiamRPN_OTB2015(OTB2015 AUC:0.5668)
did you have the better result? I  did as the author speak, why I get bad result?

**Comment by foolwood** [comment]
@csg1111 
Please try to use python 2.7.

see more in https://github.com/foolwood/DaSiamRPN/issues/39

**Comment by csg1111** [comment]
Thanks for yor  solution, I  have got  the same resulrs.
And could you serve the training code? I will be  grateful!

**Comment by csg1111** [comment]
Thanks for your solution, I have got the same results.
And could you serve the training code? I will be grateful !
